homeassistant/components/imap_email_content/__init__.py

```diff
@@ -2,10 +2,15 @@
 
 from homeassistant.const import Platform
 from homeassistant.core import HomeAssistant
+from homeassistant.helpers import config_validation as cv
 from homeassistant.helpers.typing import ConfigType
 
+from .const import DOMAIN
+
 PLATFORMS = [Platform.SENSOR]
 
+CONFIG_SCHEMA = cv.removed(DOMAIN, raise_if_present=False)
```

Comment:
I'm not entirely sure if `cv.removed` or `cv.deprecated` is correct here

Reply:
Yet it is deprecated.